lib/components/user/with-logged-in-user-support.js

```diff
@@ -37,33 +37,34 @@ export default function withLoggedInUserSupport (WrappedComponent, requireLogged
  * Upon completion (or if no user is logged in or if auth is disabled), it renders children.
  */
 class UserLoaderScreen extends Component {
+  /**
+   * Determines whether user data should be fetched.
+   * @returns true if the logged-in user has passed Auth0 authentication and state.user.loggedInUser has not been set; false otherwise.
+   */
+  shouldFetchUser = () => {
```

Comment:
Would a better name be `userIsMissing/Unfetched`?

Reply:
Good suggestion. Updated in https://github.com/opentripplanner/otp-react-redux/pull/233/commits/c9bb706107fffe6b477e73123313da05a5031f74.